functions/exercise19/scriptSpec.js

```diff
@@ -0,0 +1,44 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals returnElementsLargerThanNumber*/
+
+describe("Returning array elements larger than a number", function () {
+    it("empty array aren't any elements larger than 10", function () {
+        var elementsLargerThanANumber = returnElementsLargerThanNumber(10, []);
+        expect(elementsLargerThanANumber).toEqual([]);
+    });
+    it("in the array whit white space [ ]  aren't any elements larger than 10", function () {
+        var elementsLargerThanANumber = returnElementsLargerThanNumber(10, []);
+        expect(elementsLargerThanANumber).toEqual([]);
+    });
+    it("white space [ ] is an exception ", function () {
+        expect(function () { returnElementsLargerThanNumber(" "); }).toThrow(new Error("Parameter must contain at least one number - not use white space"));
+    });
+    it("letter is an exception ", function () {
+        expect(function () { returnElementsLargerThanNumber("a"); }).toThrow(new Error("Parameter must contain at least one number - not use letters"));
+    });
+    it("in the array [10] aren't any elements larger than 10", function () {
+        var elementsLargerThanANumber = returnElementsLargerThanNumber(10, [10]);
+        expect(elementsLargerThanANumber).toEqual([]);
+    });
+
+    it("in the array [2, 5, 10] aren't any elements larger than 10 ", function () {
+        var elementsLargerThanANumber = returnElementsLargerThanNumber(10, [2, 5, 10]);
+        expect(elementsLargerThanANumber).toEqual([]);
+    });
+    it("in the array [-2, -5, 10, 15] elements larger than 10 are [15]", function () {
```

Comment:
Please emhasize in this test description that negative values are also important here.